doc/sphinx/conf.py

```diff
@@ -337,10 +341,14 @@ def escape_splats(app, what, name, obj, options, lines):
         {
             "name": "GitHub",
             "url": "https://github.com/Cantera/cantera",
-            "icon": "fa-brands fa-square-github",
+            "icon": "fa-brands fa-github",
             "type": "fontawesome",
         }
-   ],
+    ],
+    "announcement": (
+        "This is a community-supported project. If you'd like to contribute, "
+        "<a href='https://cantera.org/getting-involved.html'>check out our 'Getting Involved' page</a>! "
```

Comment:
For the first sentence, I think being more explicit would be good:

> Cantera is a community-owned project and we need your help!

The second sentence, "contribute" doesn't sound right to me, but maybe that's just me ☺️ To actually offer a suggestion: 

> If you want to support Cantera, check out the Getting Involved page. Thank you!

Just a suggestion, you don't need to take it completely!

Reply:
@bryanwweber ... thanks, but we need to be as short as possible (otherwise it will break into two lines as the window gets narrower). Here's another version:

Cantera is community-driven - want to help? [Get involved here](https://cantera.org/getting-involved.html)!

Reply:
That makes sense. What about dropping the question so it's more active and urgent:

> Cantera is community-driven and needs your help! Get involved here

Reply:
Sounds good! Here's what we have now:

<img width="1390" alt="image" src="https://github.com/user-attachments/assets/57e1bfcb-752b-47d8-959e-7ef618b663fa" />